pennylane/ops/qubit/special_unitary.py

```diff
@@ -0,0 +1,346 @@
+        ``["I", "X", "Y", "Z"]``, in the order of the qubits
+
+      - such that for three qubits, the first Pauli words are
+        ``"IIX", ""IIY", "IIZ", "IXI", "IXX", "IXY", "IXZ", "IYI"...``
+
+    **Example**
+
+    >>> pauli_words(1)
+    ['X', 'Y', 'Z']
+    >>> len(pauli_words(3))
+    63
+    """
+    return ["".join(letters) for letters in list(product(_pauli_letters, repeat=num_wires))[1:]]
+
+
+def special_unitary_matrix(theta, num_wires):
+    r"""Compute the matrix of an element in SU(N), given by the Pauli basis coordinated
+    of the associated Lie algebra element.
+
+    Args:
+        theta (tensor_like): Pauli coordinates of the exponent :math:`A(\theta)`.
+        num_wires (int): The number of wires the matrix acts on.
+
+    Returns:
+        tensor_like: matrix of the special unitary corresponding to ``theta``. It
+            has the shape ``(2**num_wires, 2**num_wires)``.
+
+    The :math:`4^n-1` Pauli basis elements of the Lie algebra :math:`\mathfrak{su}(N)`
+    for :math:`n` qubits are
+    :math:`P_m\in\{I, X, Y, Z\}^{\otimes n}\setminus\{I^{\otimes n}\}`, and the special
+    unitary matrix is computed as
+
+    .. math::
+
+        U(\theta) = \exp(i\sum_{m=1}^d \theta_m P_m)
+
+    See :func:`~.ops.qubit.matrix_ops.pauli_words` for the ordering of Pauli words.
+
+    .. note::
+
+        Note that this method internally handles a complex-valued tensor of size
+        ``(4**num_wires, 2**num_wires, 2**num_wires)``, which requires at least
+        ``4 ** (2 * num_wires - 13)`` GB of memory (at default precision).
+    """
+    if num_wires > 5:
+        theta = qml.math.hstack([[0], theta])
+        A = sum(
+            t * reduce(np.kron, letters)
+            for t, letters in zip(theta, product(_pauli_matrices, repeat=num_wires))
+        )
+    else:
+        A = qml.math.tensordot(theta, pauli_basis(num_wires), axes=[[-1], [0]])
+    return qml.math.expm(1j * A)
+
+
+class SpecialUnitary(Operation):
+    r"""Gate from the group :math:`SU(N)` with :math:`N=2^n` for :math:`n` qubits.
+
+    We use the following parametrization of a special unitary operator:
+
+    .. math::
+
+        U(\theta) &= e^{A(\theta)}\\
+        A(\theta) &= \sum_{m=1}^d i \theta_m P_m\\
+        P_m &\in {I, X, Y, Z}^{\otimes n} \setminus \{I^{\otimes n}\}
+
+    This means, :math:`U(\theta)` is the exponential of operator :math:`A(\theta)`,
+    which in turn is a linear combination of Pauli words with coefficients :math:`i\theta`
+    and satisfies :math:`A(\theta)^\dagger=-A(\theta)` (it is *skew-Hermitian*).
+    Note that this gate takes an exponential number :math:`d=4^n-1` of parameters.
+
+    **Details:**
+
+    * Number of wires: Any
+    * Number of parameters: 1
+    * Number of dimensions per parameter: (1,)
+    * Gradient recipe
+
+    .. math::
+
+        \frac{\partial}{\partial\theta_\ell} f(U(\theta)) &= -i \sum_{m=1}^d \omega_{\ell m}
+        \frac{\mathrm{d}}{\mathrm{d} x} f(e^{ixP_m} U(\theta))
```

Comment:
Also doesnt render, maybe its the break?
```suggestion
    .. math::
        \frac{\partial}{\partial\theta_\ell} f(U(\theta)) &= -i \sum_{m=1}^d \omega_{\ell m}
        \frac{\mathrm{d}}{\mathrm{d} x} f(e^{ixP_m} U(\theta))
```

Reply:
The problem was with the indentation of the _following_ block :sweat_smile: